lib/bundler/plumber/cli.rb

```diff
@@ -33,6 +33,7 @@ class CLI < ::Thor
       desc 'check', 'Checks the Gemfile.lock for known memory leaks'
       method_option :quiet, :type => :boolean, :aliases => '-q'
       method_option :verbose, :type => :boolean, :aliases => '-v'
+      method_option :ignore, :type => :array, :aliases => '-i'
```

Comment:
@bronzdoc Any chance we can add an example value to this method option? It's not clear to me how to pass `n` gems to be ignored. It's clear how you pass one value.

Reply:
The help method for `check` has an example:
```
 -i, [--ignore=one two three]
```